### PR TITLE
Bump eks and aks operators

### PR DESCRIPTION
**Pull request by a-blender**
Pull in updates
* aks-operator: https://github.com/rancher/rancher/issues/39259
* eks-operator: https://github.com/rancher/rancher/issues/29071

### CODE REVIEW

File: release.yaml [hunk 1]
```diff
@@ -16,3 +16,15 @@ rancher-monitoring:
...
+  - 101.1.0+up1.0.8-rc1
+rancher-aks-operator-crd:
+  - 101.1.0+up1.0.8-rc1
+  - 101.1.0+up1.0.8-rc1
+rancher-eks-operator:
+  - 101.1.0+up1.1.6-rc1
+  - 101.1.0+up1.1.6-rc1
+rancher-eks-operator-crd:
+  - 101.1.0+up1.1.6-rc1
+  - 101.1.0+up1.1.6-rc1
```
Comment (jiaqiluo) [inline release.yaml]: duplicated lines?

**Review by jiaqiluo — CHANGES_REQUESTED** [review]

File: release.yaml (hunk 1 above)
Comment (a-blender) [reply release.yaml]: Ah no, I am bumping the versions of both aks and eks which each have 2 charts (a rancher-(kev2)-operator chart and a rancher-(kev2)-operator-crd chart)

Comment (jiaqiluo) [reply release.yaml]: I mean, there are duplicated versions under each section, like two `101.1.0+up1.0.8-rc1` under `rancher-aks-operator`

Comment (a-blender) [reply release.yaml]: Perks of feeling under the weather today;) thanks

**Review by jiaqiluo — APPROVED** [review]
LGTM now

**Review by kinarashah — APPROVED** [review]

**Pull request merged**